test/extended/machines/display_reboots_pod.yaml

```diff
@@ -1,13 +1,17 @@
 apiVersion: v1
 kind: Pod
 metadata:
+  annotations:
+    openshift.io/required-scc: privileged
   generateName: list-boots-
 spec:
   restartPolicy: Never
   hostPID: true
   containers:
-    - command:
-        - exec chroot /host-root journalctl --list-boots
```

Comment:
this kept resulting a "command not found" style error.  I didn't bother trying to work out why. By executing bash instead, we get the desired result.